From dbbae387736ddab7f0f303cc4dc766cc312fd619 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:04:23 -0400
Subject: [x86] perf: Export perf_assign_events()

Message-id: <1410181752-25631-19-git-send-email-jolsa@redhat.com>
Patchwork-id: 91692
O-Subject: [PATCH RHEL7.1 BZ1134356 018/307] perf/x86: Export perf_assign_events()
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit 4a3dc121d3c370625575247bf714db3f601d83e9
Author: Yan, Zheng <zheng.z.yan@intel.com>
Date: Tue Mar 18 16:56:43 2014 +0800

description
===========
export perf_assign_events to allow building perf Intel uncore driver
as module
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/kernel/cpu/perf_event.c b/arch/x86/kernel/cpu/perf_event.c
index ab5db10..91f4422 100644
--- a/arch/x86/kernel/cpu/perf_event.c
+++ b/arch/x86/kernel/cpu/perf_event.c
@@ -722,6 +722,7 @@ int perf_assign_events(struct event_constraint **constraints, int n,
 
  return sched.state.unassigned;
 }
+EXPORT_SYMBOL_GPL(perf_assign_events);
 
 int x86_schedule_events(struct cpu_hw_events *cpuc, int n, int *assign)
 {
-- 
1.7.1